provision-contest/ansible/group_vars/all/secret.yml.example

```diff
@@ -1,53 +1,53 @@
 # Password for the MySQL replication user.
 # Set this to enable master-master replication between two domservers.
-#REPLICATION_PASSWORD: some-replication-password
+#REPLICATION_PASSWORD: {some-strong-replication-password}
```

Comment:
are the `{}` part of the password?

Reply:
This is an example file, and no I use them as anchors to replace them in the script I added.

Normally we would `cp secret.yml{example,}` and then replace the passwords by manually running xkcdpass, I now tried to automate this to make sure we always use unique ones.

Reply:
Maybe we should document this?
Also we still manually need to enable replication, right?

Reply:
My question was because if someone does it manually, then this format suggests to keep the `{}` surrounding the password, which they probably should not.

Reply:
I've now documented it